src/compile/selection/selection.ts

```diff
@@ -410,3 +415,11 @@ export function positionalProjections(selCmpt: SelectionComponent) {
   });
   return {x, xi, y, yi};
 }
+
+export function assembleInit(init: any, wrap: (str: string) => string = identity): string {
```

Comment:
Why is `init` `any` here? Can we use a stricter type?

Reply:
It should be `SelectionInit | SelectionInitArray` but I'm not able to figure out why TypeScript throws a big error with the `map` call on the next line... `isArray` sets up a type guard...

Reply:
Fixed in 7f796587b